metaci/build/models.py

```diff
@@ -528,11 +528,12 @@ def load_junit(self, filename):
                 if element.tag not in ['failure', 'error']:
                     continue
                 result['Outcome'] = 'Fail'
-                result['StackTrace'] += element.text + '\n'
+                if element.text:
+                    result['StackTrace'] += element.text + '\n'
                 message = element.attrib['type']
-                if 'message' in element.attrib:
+                if 'message' in element.attrib and element.attrib['message']:
```

Comment:
I think you can just do `if element.get('message')`

Reply:
wouldn't it have to be element.attrib.get('message') or does ET provide a get method for attributes?

Reply:
https://docs.python.org/2/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.get